packages/dx-react-grid/src/plugins/local-filtering.jsx

```diff
@@ -8,7 +8,12 @@ export class LocalFiltering extends React.PureComponent {
     const { filterFn } = this.props;
 
     return (
-      <PluginContainer>
+      <PluginContainer
+        pluginName="LocalFiltering"
+        dependencies={[
```

Comment:
Typically, creating inlining a complex constant in a render method is not a good practice for performance reasons. Please, consider extracting constants.